### PR TITLE
Issue #974 Created anonymous functions

### PR DESCRIPTION
**Pull request by mmoreram**
* Instead of public functions.

### CODE REVIEW

File: src/Elcodi/Component/Metric/API/Twig/APIMetricExtension.php [hunk 1]
```diff
@@ -52,87 +52,31 @@ public function __construct(AbstractMetricsBucket $metricBucket)
...
-     */
-    public function getDistributions(
-        $token,
-        $event,
-        $dates
-    ) {
-        return (int) $this
-            ->metricBucket
-            ->getDistributions($token, $event, $dates);
+                return (int) $this
```
Comment (EmanueleMinotto) [inline src/Elcodi/Component/Metric/API/Twig/APIMetricExtension.php]: `getDistributions` return an array

File: src/Elcodi/Component/Metric/API/Twig/APIMetricExtension.php [hunk 2]
```diff
@@ -52,87 +52,31 @@ public function __construct(AbstractMetricsBucket $metricBucket)
...
-    public function getAccumulation(
-        $token,
-        $event,
-        $dates
-    ) {
-        return $this
-            ->metricBucket
-            ->getAccumulation($token, $event, $dates);
-    }
+                return $this
```
Comment (EmanueleMinotto) [inline src/Elcodi/Component/Metric/API/Twig/APIMetricExtension.php]: `getAccumulation` return an integer, so can be forced like in the closures above

**Comment by EmanueleMinotto** [comment]
@mmoreram after those two fixes, this PR will be ok for me :+1:

**Comment by mmoreram** [comment]
@EmanueleMinotto If removed all logic from the extension, and this includes all casting. When you add any `(int)` then you're creating logic, but the responsibility to add this changes is from the service itself. I've removed all logic :)

**Comment by EmanueleMinotto** [comment]
seems right, well done :)

**Pull request merged**